### PR TITLE
Add documentation to readme

### PR DESCRIPTION
**Pull request by sndrs**

### CODE REVIEW

**Review by SiAdcock — DISMISSED** [review]
Sweet as, man

File: README.md [hunk 1]
```diff
@@ -1,7 +1,152 @@
...
+```js
+import { from, until } from 'src-mq'
+
+const styles = {
+
+	[until.medium.for('print')]: {
+		color: 'green',
+	},
+
+	[from.large.for('screen and print')]: {
```
Comment (SiAdcock) [inline README.md]: This should probably be `for('screen, print')`

Comma is a [logical `or` operator](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#(comma))

See [my rant in the other PR](https://github.com/src-mq/src-mq/pull/17#discussion_r321985110)

Comment (sndrs) [reply README.md]: 👍

**Comment by sndrs** [comment]
I've updated this along the lines of what we've been discussing offline. Please feel free to pick holes!

Shall we use this PR to maybe kick the API about? once we're happy with this readme we update the tests to cover it and then the code?

**Comment by sndrs** [comment]
@SiAdcock i think is up to date with what we've discovered/decided. 

if it all worked like this, you think it's v1able?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,212 @@
...
+
+- `xxSmall` (320 pixels)
+- `xSmall` (375 pixels)
+- `small` (480 pixels)
+- `medium` (740 pixels)
+- `large` (980 pixels)
+- `xLarge` (1140 pixels)
+- `xxLarge` (1300 pixels)
+
+_Breakpoint values are output to CSS as ems, assuming 1em = 16px._
```
Comment (SiAdcock) [inline README.md:137]: I guess this should be made configurable in the long term?

**Review by SiAdcock — APPROVED** [review]
Amazing work @sndrs 🎆 The API described here looks v1able to me!

File: README.md (hunk 2 above)
Comment (sndrs) [reply README.md:137]: ems vs px etc?

**Comment by SiAdcock** [comment]
I dunno, em seems more useful than px as an output unit. I mean the assumption that 1em = 16px. Can the consumer override the body font size and break the assumption?

**Pull request merged**